docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -396,9 +396,7 @@ schema.
 .. warning::
 
    Using ``CREATE TABLE`` with an existing table content is deprecated, instead use the
-   ``system.register_table`` procedure. The ``CREATE TABLE ... WITH (location=...)``
-   syntax can be temporarily re-enabled using the ``delta.legacy-create-table-with-existing-location.enabled``
-   config property or ``legacy_create_table_with_existing_location_enabled`` session property.
+   ``system.register_table`` procedure.
```

Comment:
> Using ``CREATE TABLE`` with an existing table content is deprecated

This sentence needs to be updated. Also, there's no need to be warning anymore in my opinion. The normal sentence looks good to me.